experimental/README.md

```diff
@@ -0,0 +1,19 @@
+# ECS Experimental Definitions
+
+ECS experimental definitions are changes and features which have been merged as [stage two](https://elastic.github.io/ecs/stages.html) in the ECS [RFC process](../rfcs), but they have not yet advanced stage three. These initial field definitions are comprehensive but not necessarily complete. They are also still subject to breaking changes or being removed in any future version.
+
+This directory provides a location for these experimental field definitions and generated artifacts.
+
+## Schema Files
+
+The [schemas](./schemas) directory contains the YAML files for the experimental field definitions. These are not complete schemas themselves, but are settings meant to merge with the official schema spec using the `--include` generator flag.
+
+Example usage:
+
+```sh
+$ python scripts/generator.py --include experimental/schemas --out experimental
+```
```

Comment:
The current example is actually what we need to run to maintain this directory.

I would instead document this for folks that want to use the experimental artifacts for their own custom index. So we should refer them to USAGE.md and also give an example of grabbing experimental changes + typical customization. Here's my take on this:

> If you use the ECS generator script as described in [USAGE.md](https://github.com/elastic/ecs/blob/master/USAGE.md) to maintain your custom index templates, here's how you can try these experimental changes in your project:

```sh
$ python scripts/generator.py --include experimental/schemas \
        --include ../myproject/fields/custom/ \
        --out ../myproject/fields/generated
```

> The above would include all experimental changes to ECS along with your custom fields, and output the artifacts in `myproject/fields/generated`.

Reply:
Good call out. Also good steer users towards the usage documentation 😄